Libraries/Components/WebView/WebView.ios.js

```diff
@@ -462,6 +462,29 @@ var WebView = React.createClass({
   },
 
   /**
+   * Evaluate JavaScript on the current page.
+   */
+  evaluateJavaScript: async function(
+    script: string,
+    callback?: ?(error: ?Error, result: ?string) => void,
```

Comment:
We don't need to support both `Promise` and callback based APIs. React Native is moving to using `Promises` instead of callbacks. :)